server/server.go

```diff
@@ -21,8 +21,8 @@ type Option func(s *Server)
 // Server serves the Flipt backend
 type Server struct {
 	logger logrus.FieldLogger
-
-	store storage.Store
+	store  storage.Store
+	flipt.UnimplementedFliptServer
```

Comment:
do you know the reasoning for why this was added?

Reply:
Yeah, for some reason the code base was upgraded (thought it happened between my last PR) to use the new go protc plugin which in order to retain BC with the old package asks you to embed the unimplemented server.